compose-files/docker-compose.yml

```diff
@@ -142,7 +142,7 @@ services:
       - CONTENT_URI=https://consent.schul-cloud.org #lernstore
       - PORT=3030
       # AUTH Secret
-      - LDAP_PASSWORD_ENCRYPTION_KEY=fortestingonly12
+      - AES_KEY=fortestingonly12
```

Comment:
for a short time it's better to have both in this file.
After hpi-schul-cloud/schulcloud-server#3330 is merge it can be reduce to AES_KEY only.

Reply:
@mamutmk5 is it also ok if we wait for #[https://github.com/hpi-schul-cloud/schulcloud-server/pull/3330](hpi-schul-cloud/schulcloud-server#3330) to be merged and then do the merge here?